ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -363,6 +363,10 @@ defaults <- node_group <- node_group.node <- node
     # MLAG interfaces speed | Optional and depends on mlag_interfaces to be defined
     mlag_interfaces_speed: < interface_speed | forced interface_speed | auto interface_speed >
 
+    # MLAG peer link port-channel id: | Optional; if not set, the mlag port-channel id
+    # is generated from the digits of the first interface present in 'mlag_interfaces'
+    mlag_port_channel_id: < 1-2000 > for EOS < 4.25.0F | < 1 - 999999 > for EOS >= 4.25.0F
```

Comment:
Just a note for general awareness. Even though we can use a higher PO number, we cannot increase the MLAG id above 2000. Not relevant for this feature, but worth mentioning.
Another point on this PR. The whole reason for it, was the problem of PO > 2000 on modulars. Since as found out here >=4.25.0F can support >2000 do we still want it? It could lead to conflicts if the selected PO is the same as something generated dynamically for servers.

Reply:
any way of course we can add this knob to offer more control.